assets/js/googlesitekit/notifications/datastore/notifications.test.js

```diff
@@ -337,6 +412,88 @@ describe( 'core/notifications Notifications', () => {
 			return <div>Test notification!</div>;
 		}
 
+		describe( 'getSeenNotifications', () => {
+			it( 'should return an empty object when no notifications have been seen', () => {
+				const seenNotifications = registry
+					.select( CORE_NOTIFICATIONS )
+					.getSeenNotifications();
+
+				expect( seenNotifications ).toEqual( {} );
+			} );
+
+			it( 'should return seen notifications with their view counts', async () => {
```

Comment:
This test spec isn't accurate as that's not what the test verifies below (it checks for an array of values, not a count).

```suggestion
			it( 'should return seen notifications with their dates viewed as an array', async () => {
```